friends/src/App.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import axios from 'axios';
+
+import Friends from './components/Friends';
```

Comment:
There's an error because your Friends file is named `friend` instead of `Friend`